src/dfcx_scrapi/core/agents.py

```diff
@@ -348,6 +353,10 @@ def export_agent(self, agent_id: str, gcs_bucket_uri: str) -> str:
           gcs_bucket_uri: The Google Cloud Storage bucket/filepath to export the
             agent to in the following format:
               `gs://<bucket-name>/<object-name>`
+          environment: (Optional) CX Agent environment ID string. If not set,
+            draft environment is assumed. Format:
```

Comment:
Nit: place `draft` in all caps like `DRAFT` to match the naming convention across the platform.